src/SSHDebugPS/Docker/DockerHelper.cs

```diff
@@ -17,6 +17,87 @@ public class DockerHelper
         private const string dockerPSCommand = "ps";
         // --no-trunc avoids parameter truncation
         private const string dockerPSArgs = "-f status=running --no-trunc --format \"{{json .}}\"";
+        private const string dockerInspectCommand = "inspect";
+        private const string dockerInspectArgs = "-f \"{{json .Platform}}\" ";
+
+        public static string GetContainerPlatform(string hostname, string containerName)
+        {
+            string containerPlatform = string.Empty;
+
+            DockerCommandSettings settings = new DockerCommandSettings(hostname, false);
+            settings.SetCommand(dockerInspectCommand, string.Concat(dockerInspectArgs, containerName));
+
+            LocalCommandRunner commandRunner = new LocalCommandRunner(settings);
+
+            StringBuilder errorSB = new StringBuilder();
+            int? exitCode = null;
+
+            try
```

Comment:
I would suggest refactoring this and the below item as a lot of this code is duplicated.